packages/react-core/src/components/Popover/examples/PopoverAdvanced.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Popover, PopoverPosition, Checkbox, Button } from '@patternfly/react-core';
+
+export const PopoverAdvanced: React.FunctionComponent = () => {
+  const [position, setPosition] = React.useState(PopoverPosition.auto);
+  const [show, setShow] = React.useState(false);
+  const [keepInViewChecked, setKeepInViewChecked] = React.useState(true);
+
+  const handleKeepInViewChange = (checked: boolean) => {
+    setKeepInViewChecked(checked);
+  };
+
+  const handleProgrammaticChange = (checked: boolean) => {
+    setShow(checked);
+  };
+
+  const shouldClose = () => {
+    setShow(false);
+  };
+
+  const shouldOpen = () => {
+    setShow(true);
+  };
+
+  return (
+    <div>
+      <div>
+        <span style={{ paddingRight: '10px' }}>Popover position</span>
+        <select
+          aria-label="Popover position"
+          onChange={event => {
+            setPosition(event.target.value as PopoverPosition);
+          }}
+        >
+          {Object.values(PopoverPosition).map(position => (
+            <option key={position} value={position}>
+              {position}
+            </option>
+          ))}
+        </select>
+        <Checkbox
+          label="Flip popover if the position falls outside the view"
+          isChecked={keepInViewChecked}
+          onChange={handleKeepInViewChange}
+          aria-label="Keep in view"
+          id="check-3"
```

Comment:
Just to avoid possible duplicate ID's in the future:

```suggestion
          id="popover-advanced-check-1"
```